src/commands/stac-setup/stac.setup.ts

```diff
@@ -146,6 +143,20 @@ export function slugFromMetadata(metadata: SlugMetadata): string {
   throw new Error(`Slug can't be generated from collection as no matching category: ${metadata.geospatialCategory}.`);
 }
 
+/**
+ * Remove a trailing 'm' from a GSD value and log warning if it is present
+ *
+ * @param gsd GSD value from command line input
+ * @returns GSD without trailing 'm'
+ */
+export function formatGsd(gsd: string): string {
+  if (gsd.endsWith('m')) {
+    logger.warn(`${gsd} supplied as GSD; future supported format will require numerical value only.`);
```

Comment:
Do we have a date/ticket/issue that track this planned change?

Reply:
I think it would be good to describe what is a valid GSD value in [the description of the input parameter](https://github.com/linz/argo-tasks/blob/60501f1ea527ef629a8f5d2ce72506f7feae3c74/src/commands/stac-setup/stac.setup.ts#L44).

Reply:
This ticket is closed/"Done". How are we going to remember
> Deprecation notice in the code to remove at the next major release

?